.evergreen/config.yml

```diff
@@ -178,7 +178,7 @@ functions:
             # If this was a patch build, doing a fresh clone would not actually test the patch
             cp -R ${PROJECT_DIRECTORY}/ $DRIVERS_TOOLS
           else
-            git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
+            git clone https://github.com/kevinAlbs/drivers-evergreen-tools.git --branch D2377.update-creds $DRIVERS_TOOLS
```

Comment:
TODO: revert this change once https://github.com/mongodb-labs/drivers-evergreen-tools/pull/224 is merged.